Ideology/DefInjected/ThoughtDef/Precepts_Charity.xml

```diff
@@ -4,256 +4,256 @@
+  <CharityFulfilled_Worthwhile_ShuttleCrashRescue.stages.shuttle_survivors_helped.description>Pomohli jsme přeživším z havarovaného raketoplánu. Potřebovali naši pomoc.</CharityFulfilled_Worthwhile_ShuttleCrashRescue.stages.shuttle_survivors_helped.description>
   
   <!-- EN: accepted someone in danger -->
   <CharityFulfilled_Worthwhile_ThreatReward_Joiner.stages.accepted_someone_in_danger.label>pomoc někomu v nesnázích</CharityFulfilled_Worthwhile_ThreatReward_Joiner.stages.accepted_someone_in_danger.label>
   <!-- EN: We helped someone under threat. That was pretty good. -->
   <CharityFulfilled_Worthwhile_ThreatReward_Joiner.stages.accepted_someone_in_danger.description>Pomohli jsme někomu v nesnázích. Všem pomáháme bez rozdílu.</CharityFulfilled_Worthwhile_ThreatReward_Joiner.stages.accepted_someone_in_danger.description>
   
   <!-- EN: wanderer helped -->
-  <CharityFulfilled_Worthwhile_WandererJoins.stages.wanderer_helped.label>pomoc tulákovy</CharityFulfilled_Worthwhile_WandererJoins.stages.wanderer_helped.label>
+  <CharityFulfilled_Worthwhile_WandererJoins.stages.wanderer_helped.label>pomoc tulákovi</CharityFulfilled_Worthwhile_WandererJoins.stages.wanderer_helped.label>
   <!-- EN: We helped a desperate wanderer survive. That's a good thing. -->
-  <CharityFulfilled_Worthwhile_WandererJoins.stages.wanderer_helped.description>Pomohli jsme zoufalému tulákovy. To od nás bylo milé.</CharityFulfilled_Worthwhile_WandererJoins.stages.wanderer_helped.description>
+  <CharityFulfilled_Worthwhile_WandererJoins.stages.wanderer_helped.description>Pomohli jsme zoufalému tulákovi. To od nás bylo milé.</CharityFulfilled_Worthwhile_WandererJoins.stages.wanderer_helped.description>
   
   <!-- EN: beggars not helped -->
-  <CharityRefused_Essential_Beggars.stages.beggars_not_helped.label>nepomohli jsme žebrákům</CharityRefused_Essential_Beggars.stages.beggars_not_helped.label>
+  <CharityRefused_Essential_Beggars.stages.beggars_not_helped.label>nepomohlo se žebrákům</CharityRefused_Essential_Beggars.stages.beggars_not_helped.label>
   <!-- EN: They begged for help, but we refused. What a violation - am I even human any more? -->
-  <CharityRefused_Essential_Beggars.stages.beggars_not_helped.description>Oni žebrali o pomoc a my jsme jim nepomohli. Jsme vůbec lidi?</CharityRefused_Essential_Beggars.stages.beggars_not_helped.description>
+  <CharityRefused_Essential_Beggars.stages.beggars_not_helped.description>Oni žebrali o pomoc a my jsme jim nepomohli. Jsme ještě vůbec lidi?</CharityRefused_Essential_Beggars.stages.beggars_not_helped.description>
   
   <!-- EN: beggars betrayed -->
-  <CharityRefused_Essential_Beggars_Betrayed.stages.beggars_betrayed.label>zradili jsme žebráky</CharityRefused_Essential_Beggars_Betrayed.stages.beggars_betrayed.label>
+  <CharityRefused_Essential_Beggars_Betrayed.stages.beggars_betrayed.label>zrazení žebráků</CharityRefused_Essential_Beggars_Betrayed.stages.beggars_betrayed.label>
```

Comment:
co `zrada žebráků`? zrazení mi přijde trochu divné, ale je to jenom můj subjektivní pocit